### DISCUSSION TITLE
Media folders

### DISCUSSION THREAD

**Discussion by mojubouri** [spatie/laravel-medialibrary#2865] [Q&A]
Hello Dear,

I have a problem with the media folder in public, any files I upload it the folder will be 700 permission, and I tried to change the folder -R chmod but it's still the same problem

**Comment by PaolaRuby** [comment]
Set default permissions on config file

**Comment by mojubouri** [comment]
> Set default permissions on config file

But in other project I didn't set any default permissions and its work,

I need to understand the problem?
